infinitest-intellij/src/main/java/org/infinitest/intellij/idea/IdeaModuleSettings.java

```diff
@@ -51,6 +51,9 @@
 	private final Module module;
 	private final InfinitestJarLocator locator = new InfinitestJarLocator();
 
+  private static final Boolean TEST_CLASSES = true;
+  private static final Boolean MAIN_CLASSES = false;
```

Comment:
You're right, I tried my best not to mess up formatting.  I'll make a squash commit with the correct formatting.